siloLapisTests/test/details.spec.ts

```diff
@@ -1,11 +1,11 @@
 import { expect } from 'chai';
-import { lapisClient } from './common';
+import { basePath, lapisClient } from './common';
 import fs from 'fs';
 import { SequenceFilters } from './lapisClient';
 
 describe('The /details endpoint', () => {
   it('should return details with specified fields', async () => {
-    const result = await lapisClient.postDetails({
+    const result = await lapisClient.postDetails1({
```

Comment:
Is the "1" in `postDetails1` intetional?

Reply:
Well, I'd prefer to not have it, but it's the operation id that Springdoc assigns to the controller method. The "1" appeared, since it's overloaded.

Reply:
This is already the "nice version", since I've manually overwritten the operation id. Without it, it looked even weirded. But in the end, this is not really important, since it doesn't affect any functionality. The openapi generator just generates methods with weird names.